src/main/resources/lib/credentials/select.jelly

```diff
@@ -71,8 +75,9 @@
                 default="${paramDefault?'':attrs.default}" value="${paramValue?(paramDefault?'':attrs.default):value}"/>
       <!-- TODO  add support for checking permissions against stores in request path -->
       <st:nbsp/>
+      <j:set var="storeItems" value="${selectHelper.getStoreItems(context, includeUser)}"/>
       <j:choose>
-        <j:when test="${h.hasPermission(selectHelper.CREATE)}">
+        <j:when test="${h.hasPermission(selectHelper.CREATE) and storeItems != null and !storeItems.isEmpty()}">
```

Comment:
@oleg-nenashev not quite, if the resolution fails we will have an empty list so the button should be disabled